pkg/config/config_test.go

```diff
@@ -319,23 +319,38 @@ func (s *MySuite) TestNewBlueprint(c *C) {
 	dc := getDeploymentConfigForTest()
 	outFile := filepath.Join(tmpTestDir, "out_TestNewBlueprint.yaml")
 	dc.ExportBlueprint(outFile)
-	newDC := NewDeploymentConfig(outFile)
+	newDC, _ := NewDeploymentConfig(outFile)
```

Comment:
You should add a `c.Assert(err, IsNil)` here and anywhere else that's now returning an error.